build_tools/packaging/linux/packaging_utils.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import json
+
+def read_package_json_file():
+    '''Reads a JSON file and returns the parsed data.'''
+    with open("package.json", 'r') as file:
+        data = json.load(file)
+    return data
+
+
+def is_packaging_disabled(package):
```

Comment:
nit: could add a type hint for the `package` argument (same for other functions)